src/hipscat_import/margin_cache/margin_cache_map_reduce.py

```diff
@@ -90,13 +92,16 @@ def _to_pixel_shard(data, margin_threshold, output_path, ra_column, dec_column):
                 PartitionInfo.METADATA_PIXEL_COLUMN_NAME: np.uint64,
             }
         )
+        final_df = final_df.set_index(HIPSCAT_ID_COLUMN).sort_index()
```

Comment:
This means the margin cache now has the hipscat_index, right?

Reply:
Yes - future margin caches will carry through the `_hipscat_index` of their primary catalog.